Kajabi/sage-lib
# [SAGE-797]: Form element spacing and color adjustments spec alignment

## Description
Label, message, and colors for the following form components need to be updated for better alignment with design specs.

- Form input
- Form select
- Form textarea

The color changes are to better help differentiate when a form input is displaying placeholder text vs containing actual inputted text.


## Screenshots
FORM INPUT:
||  Before  |  After  |
|--------|--------|--------|
| Label |![Screen Shot 2022-09-07 at 2 43 01 PM](https://user-images.githubusercontent.com/1175111/188989133-de4b4645-bf43-4038-9b73-370e98216ecf.png)|![Screen Shot 2022-09-07 at 2 43 26 PM](https://user-images.githubusercontent.com/1175111/188989148-5ade170b-f70f-4cdf-bd87-8c3e0aaaec2a.png)|
|Message|![Screen Shot 2022-09-07 at 2 53 37 PM](https://user-images.githubusercontent.com/1175111/188990480-38152bc4-b34b-4170-9107-d7c1042e607d.png)|![Screen Shot 2022-09-07 at 2 53 52 PM](https://user-images.githubusercontent.com/1175111/188990518-8fb87dcf-787f-4da9-adde-e8fa23393c06.png)|
|Color (when text entered)|![Screen Shot 2022-09-07 at 2 55 15 PM](https://user-images.githubusercontent.com/1175111/188991676-5ad3d199-bcf3-49b2-8a8d-60070d234f44.png)|![Screen Shot 2022-09-07 at 2 55 39 PM](https://user-images.githubusercontent.com/1175111/188991698-687e8a9b-382e-47b7-9175-01513119414e.png)|


FORM SELECT:
||  Before  |  After  |
|--------|--------|--------|
|Label|![Screen Shot 2022-09-07 at 3 04 52 PM](https://user-images.githubusercontent.com/1175111/188992005-cc30a2b2-64c5-42bd-a5bf-a1772e36d3d2.png)|![Screen Shot 2022-09-07 at 3 05 14 PM](https://user-images.githubusercontent.com/1175111/188992031-83c23827-3706-41d4-b332-47d83f2895e3.png)|
|Color (when text entered)|![Screen Shot 2022-09-07 at 3 06 18 PM](https://user-images.githubusercontent.com/1175111/188993706-04c2f56d-2ab3-44cb-a4e2-e30359fd700c.png)|![Screen Shot 2022-09-07 at 3 06 31 PM](https://user-images.githubusercontent.com/1175111/188993735-9b24b16f-db54-46c4-84bd-8dc7ab703d61.png)|

FORM TEXTAREA:
||  Before  |  After  |
|--------|--------|--------|
|Label|![Screen Shot 2022-09-07 at 3 22 08 PM](https://user-images.githubusercontent.com/1175111/188994268-7c806b19-fc3b-44af-ba2a-918ba68b7aaf.png)|![Screen Shot 2022-09-07 at 3 24 13 PM](https://user-images.githubusercontent.com/1175111/188994309-f493558b-0ad1-4861-b9c4-e89997fb0f79.png)|
|Color (when text entered)|![Screen Shot 2022-09-07 at 3 30 48 PM](https://user-images.githubusercontent.com/1175111/188995076-a9102b95-44e8-4169-8ca2-7f84aa6c9116.png)|![Screen Shot 2022-09-07 at 3 31 20 PM](https://user-images.githubusercontent.com/1175111/188995098-00c867e6-fd3d-4ba2-bffb-58be9f4572a4.png)|


KP EXAMPLE:
|  Before  |  After  |
|--------|--------|
|![Screen Shot 2022-09-07 at 4 08 43 PM](https://user-images.githubusercontent.com/1175111/188998942-871a6dc4-d5f2-4982-8994-f95da5c22857.png)|![Screen Shot 2022-09-07 at 4 07 13 PM](https://user-images.githubusercontent.com/1175111/188998969-9ef9e667-e432-4d9a-b61d-f68f6f30e7da.png)|

## Testing in `sage-lib`
- Navigate to form elements outlined above
- Verify spacing and color adjustments align w/ design spec


## Testing in `kajabi-products`

1. (**LOW) Updates spacing and colors related to form elements to match design specs. Styling only updates.


## Related
https://kajabi.atlassian.net/browse/SAGE-797
